apps/entry/mutations.py

```diff
@@ -349,6 +377,8 @@ def mutate(root, info, id):
         figure.approved_by = info.context.user
         figure.approved_on = timezone.now()
         figure.save()
+        # NOTE: To refresh event
```

Comment:
Maybe add more detail.
```suggestion
        # NOTE: Event is updated on figure save using signals. We are clearing figure and its reference objects (event).
```

Reply:
@thenav56  Added note